integration/commands_administration_test.go

```diff
@@ -300,6 +300,45 @@ func TestCommandsAdministrationListCollections(t *testing.T) {
 
 	require.Greater(t, len(targetCollections), 2)
 
+	filter := bson.D{{
```

Comment:
I've actually renamed TestCommandsAdministrationListCollections as TestCommandsAdministrationListCollectionNames, and added a new test (which is very close to the old one but calls the ListCollectionNames method rather than ListCollections). This is why git is showing as changed, but the original test is still there (see the beginning of the new test after this block).